### DISCUSSION TITLE
clone-backend non-interactively

### DISCUSSION THREAD

**Discussion by roundrobinjp** [aws/aws-cli#7449] [Q&A] [answered]
I have a amplify backend with S3AndCloudFront hosting.
I have to clone this backend from commandline non-interactively.

With amplify cli, I can do it interactively using "amplify env add" command.
But as I said I need to do it non-interactively and
I could not find any way to do that.

So I am trying to do it using aws cli as follows:

aws amplifybackend clone-backend --app-id myappid --backend-environment-name baseenvname --target-environment-name targetenvname

It does create the target backend, but without S3AndCloudFront.

What do I need to do to achieve this?

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @roundrobinjp, thanks for reaching out.  This [documentation](https://docs.aws.amazon.com/sumerian/latest/tutorials/gltf-viewer-amplify-private.html) has information about hosting with S3AndCloudfront.  Please let me know if it's not quite what you're looking for, or if you have any other questions.
